monasca/templates/alarms-init-job.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "alarms.fullname" . }}-init-job
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Values.alarms.name }}-init-job"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        component: "{{ .Values.alarms.name }}-init-job"
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: {{ template "name" . }}-{{ .Values.alarms.name }}-init-job
+          image: "{{ .Values.alarms.image.repository }}:{{ .Values.alarms.image.tag }}"
+          imagePullPolicy: {{ .Values.alarms.image.pullPolicy }}
+          env:
+            - name: MONASCA_WAIT_FOR_API
+              value: "{{ .Values.alarms.wait.enabled }}"
+            - name: MONASCA_API_WAIT_RETRIES
+              value: "{{ .Values.alarms.wait.retries }}"
+            - name: MONASCA_API_WAIT_DELAY
+              value: "{{ .Values.alarms.wait.delay }}"
```

Comment:
you might want to convert the ints to strings here. Dont mean to be nit pick i have just seen helm be very touchy coming to this and debugging it later is like the most fun thing in the world